core/src/test/scala/org/apache/spark/ui/UISeleniumSuite.scala

```diff
@@ -48,24 +48,24 @@ import org.apache.spark.util.CallSite
 
 private[spark] class SparkUICssErrorHandler extends DefaultCssErrorHandler {
 
-  private val cssWhiteList = List("bootstrap.min.css", "vis-timeline-graph2d.min.css")
+  private val cssExcludeList = List("bootstrap.min.css", "vis-timeline-graph2d.min.css")
```

Comment:
same here instead of `exclude` how about `allowed`?

Reply:
also it seems in some cases we use `exclude` for both `whitelist` as well as `blacklist`. Like here `exclude` is used for `whiteList` and [here](https://github.com/apache/spark/blob/8f414bc6b4eeb59203ecb33c26c762a57bf5429e/resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala#L542) for `blackList`. In general, I prefer `allowedList` for `whitelist` and `denyList` or `rejectList` or `stopList` etc for `blacklist` makes it easier to comprehend quickly. I understand its hard to use the same word everywhere because of the context.

Reply:
I think `exclude` is more appropriate here as we are excluding errors from these files from the logs. Nothing about these files is being allowed. See the original PR #5983 for the intent.

I think, rather than looking at the previous name "whitelist" and trying to come up with something that matches that description, we should be looking at the variable's use/purpose and asking "what is the ideal name for this variable?". In this case, though it was referred to as a whitelist, it was really an exclude-list or an ignored list.

Reply:
I did add a comment explaining this variable to hopefully make the intent more clear.

Reply:
I agree. the previous name `whitelist` probably itself was not apt.